js/mage/adminhtml/grid.js

```diff
@@ -947,6 +947,7 @@ class varienGridAdvanced {
     }
 
     enableColumnsRearrangement() {
+        document.getElementById(this.grid.containerId + '_table').setAttribute('style', 'margin-top: 10px');
```

Comment:
why this? it adds a white space in legacy theme
![why](https://github.com/OpenMage/magento-lts/assets/5071467/40b289fb-df0b-425a-9b29-bc5a04fb263a)

Reply:
it is to move the "triangle" simbol above the grid, instead of below the heading. because below the headers it get hidden but the column that's being dragged so it's not visible

Reply:
<img width="788" alt="Screenshot 2023-09-18 alle 11 04 20" src="https://github.com/OpenMage/magento-lts/assets/909743/422efc1a-2fd7-450a-b172-c93c20985873">

Reply:
cause you have changed the arrow to be displayed on top, if you not like the bottom arrow we can add a vertical line
like this:
![](https://github.com/OpenMage/magento-lts/assets/5071467/2ac30f17-f8de-4ab8-8c56-e612cfd10e7f)

Reply:
as I wrote before, if you have it on the bottom, it gets hidden by the whole column being dragged and it's not visible.

maybe the vertical line is ok, it it's visible during dragging.

Reply:
done!